Admin/AbstractAdmin.php

```diff
@@ -3063,7 +3072,7 @@ protected function configureRoutes(RouteCollection $collection)
       *
       * @return array
       */
-     protected function configureBatchActions($actions)
+     protected function configureBatchActions(&$actions)
```

Comment:
We already use by ref calls for some others methods.

Do you have any idea how to fix this in a BC way?